homeassistant/components/goodwe/select.py

```diff
@@ -38,24 +47,27 @@ async def async_setup_entry(
     inverter = hass.data[DOMAIN][config_entry.entry_id][KEY_INVERTER]
     device_info = hass.data[DOMAIN][config_entry.entry_id][KEY_DEVICE_INFO]
 
+    supported_modes = await inverter.get_operation_modes(False)
+    OPERATION_MODE.options = [
```

Comment:
Don't modify the constant description. Pass the discovered modes separately to the entity.

Reply:
Done.